assets/scripts/game.js

```diff
@@ -0,0 +1,131 @@
+const Player = () => {
+    let marker = "X"
+    const getMarker = () => marker
+    const switchPlayer = () => {
+        if(marker === "X") {
+            marker = "O"
+        } else {
+            marker = "X"
+        }
+    }
+    return { marker, switchPlayer, getMarker }
+}
+
+const GameBoard = (() => {
+    let moves = ["", "", "", "", "", "", "", "", ""]
+    const winningCombination = [
+                                    [0,1,2],
+                                    [3,4,5],
+                                    [6,7,8],
+                                    [0,4,8],
+                                    [1,2,3],
+                                    [0,3,6],
+                                    [1,4,7],
+                                    [2,5,8],
+                                    [2,4,6]
+                                ]
+    const currentPlayer = Player()
```

Comment:
Instead of instantiating a Player object inside board, the more proper implementation would be to just pass an instantiated player onto `GameBoard`. Why? Suppose there are changes in how we instantiate Player later on, then we would also have to figure out how to tweak GameBoard.